### DISCUSSION TITLE
How to disable all services except Suricata and RDP

### DISCUSSION THREAD

**Discussion by myrsecurity** [Security-Onion-Solutions/securityonion#7263] [Unsupported Versions] [answered]
Hi Security Onion team, the product is amazing and provides a wealth of information for our security teams. 

One of our customers needs to capture only RDP and Suricata information in Kibana. Is it possible to disable the rest of the capturing services? such as conns, SSH, DHCP, DNS etc..  Reason is storage economics they want to streamline the protocol detection 

Thanks so much

**Comment by mnasec** [comment, marked as answer]
I would suggest using BPF for the different services, see documentation: https://docs.securityonion.net/en/2.3/bpf.html?highlight=bpf#bpf. Filter could be : 

```
steno:
 bpf:
  - port 3389

nids:
 bpf:
  - port 3389

zeek:
 bpf:
  - port 3389
 
```
So SO will only do the analyse for port 3389 (RDP) traffic, i think.
For space reason i set a bpf to exclude port 443 traffic only for steno, but use the nids (suricata) and zeek data. It is up to you.
